pennylane/operation.py

```diff
@@ -2054,13 +2084,14 @@ def __mul__(self, a):
 
     __rmul__ = __mul__
 
-    def __sub__(self, other):
+    def __sub__(self, other: "Observable") -> "Observable":
```

Comment:
```suggestion
    def __sub__(self, other: Operator) -> Operator:
```

We can probably go back and update the docstring at some point.